userguide/storage.rst

```diff
@@ -1280,6 +1289,51 @@ changed, and destroying a zvol requires confirmation.
 Managing Encrypted Volumes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+%brand% automatically generates a randomized **encryption key** 
+whenever a new encrypted volume is created. The data "at rest" (on disk)
+on an encrypted volume is **always encrypted**, but access to the decrypted
+data on a live (running) system can be customized for each volume.
+
+By default, encryption keys will be stored locally within %brand*'s data 
+files. They can also be downloaded as a safety measure, to allow
+decryption on a different system in the event of failure, or to allow
+the locally stored key to be deleted for extra security. Encryption keys
+can also be optionally protected with a **passphrase**. The security 
+implications are:
+
+* *Key stored locally, no passphrase* - data is automatically decrypted 
+  and always accessible when system running. (Protects "data at rest" only.)
+
+* *Key stored locally, with passphrase* - user must provide passphrase
+  before anyone can access data.
+
+* *Key not stored locally* - user must provide key before anyone can access
+  data. If a passphrase is set, this must **also** be provided before data 
+  can be accessed (`two factor authentication <https://en.wikipedia.org/wiki/Multi-factor_authentication>`_).
+
+Decrypted data **cannot be accessed** (other than from *L2ARC*) when the
```

Comment:
The reader will ask "how can I access my lost data in the L2ARC?", so I would remove that aside.  Incidentally, asides should be avoided as an interruption in the sentence.

Reply:
The problem is that, if that isn't made very clear indeed, then a user will get the message "decrypted data cannot be accessed", and probably not link it to the mention some distance up, that L2ARC isn't ever encrypted. Especially given the price of large SSD's, a user who buys any L2ARC is likely to have quite a lot of it (and 250GB will be quite a lot from a "leakable data" perspective). So the reality is that a lot of the pool's most commonly/recently accessed files will present _no encryption barrier at all_. The docs need to be very clear/explicit on that point in the section for users considering encrypted volumes, in the context of any statement suggesting that data at rest is secure. 

If the concern is _"user with data loss may wonder if lost data can be recovered from their L2ARC"_, then maybe there's a better way to say "not really" in the "lost data" section of the guide?